source/code/source/playfab/PlayFabEventPipeline.cpp

```diff
@@ -335,7 +335,7 @@ namespace PlayFab
         }
     }
 
-    void PlayFabEventPipeline::CallbackRequest(std::shared_ptr<const IPlayFabEmitEventRequest> request, std::shared_ptr<const IPlayFabEmitEventResponse> response)
+    void PlayFabEventPipeline::CallbackRequest(const std::shared_ptr<const IPlayFabEmitEventRequest>& request, const std::shared_ptr<const IPlayFabEmitEventResponse>& response)
```

Comment:
disagree

Reply:
Since this is based on a public facing callback function declaration that I stated in an earlier comment that we shouldn't change, then yeah we can revert this.